packetbeat/protos/dhcpv4/_meta/fields.yml

```diff
@@ -186,6 +186,7 @@
                 IP address.  In a server reply (DHCPOFFER), a DHCP server uses
                 this option to specify the lease time it is willing to offer.
 
+            # error.message? or just message?
```

Comment:
Yeah I vote for `error.message`, although if there's no time, not the end of the world.